{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/static_source/css/base/_forms.scss

```diff
@@ -36,6 +38,19 @@ $background-color: #fff;
     background-color 0s ease-out 200ms;
 }
 
+// Neither of these animations have a visible effect - they are used as flags for autofill
```

Comment:
did you find out how to do this somewhere? it would be cool to include a link

Reply:
Yup! There's a breakdown here that I'll add a link to in the docs
https://stackoverflow.com/a/41530164/21726578

Reply:
Aaand we have an actual breakdown here! https://medium.com/@brunn/detecting-autofilled-fields-in-javascript-aed598d25da7